contracts/LRTDepositPool.sol

```diff
@@ -259,11 +271,46 @@ contract LRTDepositPool is ILRTDepositPool, LRTConfigRoleChecker, PausableUpgrad
         address nodeDelegator = nodeDelegatorQueue[LST_NDC_INDEX];
 
         // Claim the withdrawal from the NodeDelegator
-        (asset, assets) = INodeDelegatorLST(nodeDelegator).claimWithdrawal(withdrawal, msg.sender);
+        (asset, assets) = INodeDelegatorLST(nodeDelegator).claimWithdrawal(withdrawal, msg.sender, msg.sender);
 
         emit WithdrawalClaimed(msg.sender, asset, assets);
     }
 
+    /// @notice PrimeETH staker claims the withdrawal of their previously requested OETH
+    /// but instead of receiving OETH, the OETH is deposited into Yield Nest and the withdrawer receives ynLSDe tokens.
+    /// Must wait `minWithdrawalDelayBlocks` on EigenLayer's `DelegationManager` contract
+    /// before claiming the withdrawal.
+    /// This is currently set to 50,400 blocks (7 days) on mainnet. 10 blocks on Holesky.
+    /// @dev The asset is validated against the withdrawal strategy in EigenLayer's `StrategyBase`.
+    /// @return ynLSDeAmount the amount of ynLSDe tokens received after OETH is deposited into Yield Nest
+    function claimWithdrawalYn(
+        IDelegationManager.Withdrawal calldata withdrawal
+    )
+        external
+        whenNotPaused
+        nonReentrant
+        returns (uint256 ynLSDeAmount)
+    {
+        // Get the NodeDelegatorLST contract to request the withdrawal from
+        address nodeDelegator = nodeDelegatorQueue[LST_NDC_INDEX];
+
+        // Claim the withdrawal of OETH from the NodeDelegatorLST
+        (, uint256 oethAmount) = INodeDelegatorLST(nodeDelegator).claimWithdrawal(withdrawal, msg.sender, address(this));
+
+        // Convert to Wrapped OETH (wOETH)
+        // Approve the wOETH to spend the OETH
+        IERC20(WITHDRAW_ASSET).approve(wOETH, oethAmount);
```

Comment:
nit: A `WITHDRAW_ASSET == address(OETH)` check here would be nice. But it's probably not necessary to make that change now since OETH is the only supported asset right now anyway

Reply:
`WITHDRAW_ASSET` is an immutable so can't be changed after deployment.